particula/aerosol_dynamics/particle.py

```diff
@@ -264,26 +264,124 @@ def collision_kernel_kinetic_limit(
         )
         return np.sqrt(8 * np.pi) * diffusive_knudsen_number
 
-    # # Gopalkrishnan and Hogan, 2012
-    # # doi: https://doi.org/10.1103/PhysRevE.85.026410
-    # # Equation 18
-    # def dimensionless_coagulation_kernel_GopalkrishnanHogan2012(self, other):
-    #     diffusive_knudsen_number = self.diffusive_knudsen_number(other)
-    #     continuum_limit = self.collision_kernel_continuum_limit(other)
-    #     coulomb_potential_ratio = self.coulomb_potential_ratio(other)
-    #     # kinetic_limit = self.collision_kernel_kinetic_limit(other)
-
-    #     return continuum_limit / (
-    #         1 + 1.598 * np.minimum(diffusive_knudsen_number,
-    #         3*diffusive_knudsen_number / (2*coulomb_potential_ratio)
-    #         )**1.1709
-    #     )
-
-    # # Gatti and Kortshagen 2008
-    # # doi: https://doi.org/10.1103/PhysRevE.78.046402
-    # # Retrieved from Gopalkrishnan and Hogan, 2012,
-    # # 10.1103/PhysRevE.85.026410,
-    # # Equation 13
-    # def dimensionless_coagulation_kernel_GattiKortshagen2008(self, other):
-    #     kernel_hard_sphere =
-    # self.dimensionless_coagulation_kernel_hard_sphere(other)
+    @u.wraps(u.dimensionless, [None, None, None, None])
+    def dimensionless_coagulation_kernel_parameterized(
+        self,
+        other,
+        environment: Environment,
+        authors: str = "gh2012",
+    ) -> float:
+        """Dimensionless particle--particle coagulation kernel.
+        Checks units: [dimensionless]
+
+        Paramaters:
+            self:           particle 1
+            other:          particle 2
+            environment:    environment conditions
+            authors:        authors of the parameterization
+                - gh2012    https://doi.org/10.1103/PhysRevE.78.046402
+                - cg2020    https://doi.org/XXXXXXXXXXXXXXXXXXXXXXXXXX
+                - hs        https://doi.org/XXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
add that hs: hard sphere approximation. 
it took a minute for me to figure out what that option was :)

Reply:
or call it hard_sphere